src/core/utils/BrowserFileSystem.ts

```
@@ -0,0 +1,221 @@
+        isDirectory: () => childEntry?.isDirectory || false,
+        isFile: () => !childEntry?.isDirectory,
+      };
+    });
+  }
+
+  // Read file contents
+  static async readFile(
+    filePath: string,
+    options?: { encoding?: string },
+  ): Promise<string | Uint8Array> {
+    const path = this.resolvePath(filePath);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)